<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentSettings</class>
 <widget class="QDialog" name="DocumentSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="sbCFLNumerRKScheme">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Grid spacing</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="sbMaxFreq">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>171.500000000000000</double>
     </property>
     <property name="maximum">
      <double>17150.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Max frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="sbGridSpacing">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Patch error</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Window size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Render time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Advanced</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>PML cells</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Attenuation of PML cells</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Sound speed</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Density of air</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>CFL numer RK Scheme</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Save every n-th frame</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="sbSoundSpeed">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>300.000000000000000</double>
     </property>
     <property name="maximum">
      <double>400.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="sbSaveEveryNth">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="sbAttenuationOfPMLCells">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="sbDensity">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="sbPMLCells">
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="sbRenderTime">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbWindowSize">
     <property name="minimum">
      <number>16</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Basic settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="sbPatchError">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>47.140000000000001</double>
     </property>
     <property name="maximum">
      <double>115.709999999999994</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DocumentSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocumentSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
